core/src/main/java/org/apache/accumulo/core/util/threads/ThreadPools.java

```diff
@@ -189,6 +189,10 @@ public static ThreadPoolExecutor createThreadPool(int coreThreads, int maxThread
     return result;
   }
 
+  /*
+   * If you need the server-side shared ScheduledThreadPoolExecutor, then use
+   * ServerContext.getSharedGenericScheduledExecutorService()
+   */
   public static ScheduledThreadPoolExecutor
       createGeneralScheduledExecutorService(AccumuloConfiguration conf) {
```

Comment:
Were all calls this replaced w/ the new context method or are there still some direct calls to it?

Reply:
Not all calls were replaced w/ the new context method. There are places in the code where the new method `ThreadPools.createGeneralScheduledExecutorService` did not replace the use of SimpleTimer. For example, in the [TabletServerBatchWriter](ThreadPools.createGeneralScheduledExecutorService). For this change I went back through diff of commit e62ace6a9d37572d95999f5412c6148efbba50b9 and used the new context method in places where SimpleTimer was being used.